test/1_ethereum/harvest.finance/HarvestFinanceAdapter.ts

```diff
@@ -6,12 +6,29 @@ import { HarvestFinanceAdapter } from "../../../typechain/HarvestFinanceAdapter"
 import { TestDeFiAdapter } from "../../../typechain/TestDeFiAdapter";
 import { LiquidityPool, Signers } from "../types";
 import { shouldBehaveLikeHarvestFinanceAdapter } from "./HarvestFinanceAdapter.behavior";
-import { default as HarvestFinancePools } from "./harvest.finance-pools.json";
+import { default as HarvestExports } from "@optyfi/defi-legos/ethereum/harvest.finance/contracts";
 import { IUniswapV2Router02 } from "../../../typechain";
 import { getOverrideOptions } from "../../utils";
 
 const { deployContract } = hre.waffle;
 
+const HarvestFinancePools = {
+  dai: {
+    pool: HarvestExports.harvestV1Pools.dai.pool,
+    stakingPool: HarvestExports.harvestV1Pools.dai.stakingVault,
+    lpToken: HarvestExports.harvestV1Pools.dai.lpToken,
+    tokens: HarvestExports.harvestV1Pools.dai.tokens,
+    rewardTokens: ["0xa0246c9032bC3A600820415aE600c6388619A14D"],
+  },
+  usdt: {
+    pool: HarvestExports.harvestV1Pools.usdt.pool,
+    stakingPool: HarvestExports.harvestV1Pools.usdt.stakingVault,
+    lpToken: HarvestExports.harvestV1Pools.usdt.lpToken,
+    tokens: HarvestExports.harvestV1Pools.usdt.tokens,
+    rewardTokens: ["0xa0246c9032bC3A600820415aE600c6388619A14D"],
+  },
+};
+
```

Comment:
This can be huge for protocols like Sushiswap. Would it be okay anyways?

Reply:
The above declaration is an example and I extracted just two pools. If I would be testing pools of harvest i would just use all of `HarvestExports.harvestV1Pools` and `HarvestExports.harvestV2Pools`. If that makes sense please resolve this conversation